cinema/views.py

```diff
@@ -1,17 +1,23 @@
 from datetime import datetime
 
 from django.db.models import F, Count
+from drf_spectacular.utils import extend_schema, OpenApiParameter
```

Comment:
The import statement for `OpenApiParameter` is correct, but ensure that it is used consistently throughout the file for defining API parameters.